tests/system/common.py

```diff
@@ -39,6 +40,19 @@
 marathon_1_7 = pytest.mark.skipif('marathon_version_less_than("1.7")')
 
 
+def require(requirement, message=None):
```

Comment:
Nice! I think we want to include this in the shakedown library. Unfortunately, it is not structured yet.